crmsh/corosync.py

```diff
@@ -386,6 +386,15 @@ def add_node_ucast(IParray, node_name=None):
     f = open(conf()).read()
     p = Parser(f)
 
+    # to check if the same IP already configured
+    exist_iplist = []
+    for path in p.all_paths():
+        if re.search('nodelist.node.ring[0-9]*_addr', path):
+            exist_iplist += p.get_all(path)
+    for ip in IParray:
+        if ip in set(exist_iplist):
```

Comment:
I don't really think that we need the `set` method here, but I guess is not harmfull

Reply:
Thanks!
Changed